mesonbuild/mlog.py

```diff
@@ -24,10 +25,14 @@
 log_file = None
 log_fname = 'meson-log.txt'
 
-def initialize(logdir):
```

Comment:
Not part of this patch, but the line continuation in `colorize_console =` above should be replaced with parentheses around the multi-line expression.